js/script.js

```diff
@@ -0,0 +1,53 @@
+var data ={};
+data.name="Erick González";
+data.job="Web Developer";
+data.mobile="3262-2228";
+data.email="betto793@gmail.com";
+data.github="BettoGC";
+data.location="Villa Nueva, Guatemala";
+data.information="I have the knowledge and experience in web development, front-end, and the learned languages are Html, Css, JavaScript, AnguarJS, Bootstrap, etc.";
+data.secondInfo="We learned how to develop Web pages's, also to how to develop us as people, taught us to fight for our dreams and to fulfill our goals and that we have to be able to learn, sabrena tools to be able to play in the field of programming, always working as a team and helping each other and taught us to learn from our mistakes and not to give up.";
+data.proyects="Frontend-AngularJS-Lastfm, Frontend-Pokedex-Parse, Frontend-Music, Frontend-Jasmine, Frontend-Weather-Forecast, Frontend-Blog, Frontend-Portafolio, Python-Batalla-Naval, Python-Guess-Number, Python-Maquina-Registradora, Python-Factorial, Python-Search-Engine, Python-Paises-Capitales.";
+data.skills="Respect";
+data.skills1="Responsibility";
+data.skills2="Punctuality";
+data.skills3="Fellowship";
```

Comment:
+var data ={};
+data.name="Erick González";
+data.job="Web Developer";
+data.mobile="3262-2228";
+data.email="betto793@gmail.com";
+data.github="BettoGC";
+data.location="Villa Nueva, Guatemala";
+data.information="I have the knowledge and experience in web development, front-end, and the learned languages are Html, Css, JavaScript, AnguarJS, Bootstrap, etc.";
+data.secondInfo="We learned how to develop Web pages's, also to how to develop us as people, taught us to fight for our dreams and to fulfill our goals and that we have to be able to learn, sabrena tools to be able to play in the field of programming, always working as a team and helping each other and taught us to learn from our mistakes and not to give up.";
+data.proyects="Frontend-AngularJS-Lastfm, Frontend-Pokedex-Parse, Frontend-Music, Frontend-Jasmine, Frontend-Weather-Forecast, Frontend-Blog, Frontend-Portafolio, Python-Batalla-Naval, Python-Guess-Number, Python-Maquina-Registradora, Python-Factorial, Python-Search-Engine, Python-Paises-Capitales.";
+data.skills="Respect";
+data.skills1="Responsibility";
+data.skills2="Punctuality";
+data.skills3="Fellowship";

Para que sea mas entendible tu codigo documentar y ser mas especifico en que se hace.